### ISSUE TITLE
New feature suggestion

### ISSUE THREAD

**Issue by BharZInstein** [Splode/pomotroid#173]
Just wanted to let you know that the app is just fantastic.. Great work.. I would like to also suggest a  small new feature. If the app was able to display total focused time it would be great :)

**Comment by xinfei150154** [comment]
I would like to also suggest a small new feature. If the app was able to display auto open full screen on relax it would be usefull.I want the app to give me a relaxation reminder when I forget to work by way of a full screen desktop. Then the relaxation time ends by minimizing the window to remind me to start concentrating on my work.
I think this is easy to implement in the existing code, please consider this suggestion, thanks.

**Comment by xinfei150154** [comment]
Of course, if some people don't need this feature, there is an option to turn it off

**Comment by Splode** [comment]
@BharZInstein [v1.0.0](https://github.com/Splode/pomotroid/releases/tag/v1.0.0) adds statistics visualization for daily, weekly, and yearly views of focus time.

**Comment by BharZInstein** [comment]
damn, after 5 years lets gooo!!
